docs/hub/repositories.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Repositories
+---
+
+<h1>Repositories</h1>
+
+
+## What is a repository?
+
+Models, Spaces, and datasets are hosted on the Hugging Face Hub as [Git repositories](https://git-scm.com/about), which means that version control and collaboration are core elements of the Hub. In a nutshell, a repository (also known as a **"repo"**) is a place where code and assets can be stored to back up your work and to (optionally) share it with the community. There are specific sections in these docs detailing how **models, spaces, and datasets** can each be managed, but in this document we will go over the basics of getting started with Git and interacting with repositories on the Hub. Once you have the hang of it, you can explore the best practices and next steps that we've compiled for effective repository usage.
+
+
+## Getting started
+
+### Requirements
+
+If you do not yet have `git` available as a CLI command, you will first need to [install Git for your platform](https://git-scm.com/downloads). You will also need to [install Git LFS](https://git-lfs.github.com/), which will be used to handle large files such as images and model weights.
+
+We also recommend you create a [Hugging Face account](https://huggingface.co/join) which you'll be able to use to create and manage repositories on the Hub.
```

Comment:
Yeah it's obvious, but I'd added that in just because if I talk about `huggingface-cli login`, I figured it's useful to remind them that they need to create an account if they haven't already. But I can definitely remove it!